index.js

```diff
@@ -110,11 +110,11 @@ client.on('message', async msg => {
 		return
 	}
 
-	// Prevent commands below this from being run after hours
+	// Prevent commands below this from being run outside of mentoring hours
 	let now = new Date();
-	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() != 5) {
+	if (now.getHours() < 9 || now.getHours() > 17 || now.getDay() > 5) {
```

Comment:
Only Saturday?

Reply:
This was stupid of me and has been changed.